.travis.yml

```diff
@@ -176,6 +177,17 @@ jobs:
     - GCP_BUCKET=geosx/Pangea2
     - ENABLE_HYPRE=ON
     - ENABLE_TRILINOS=OFF
+  - stage: builds
+    name: Sherlock CPU (centos 7.9.2009, gcc 10.1.0, open-mpi 4.1.0, openblas 0.3.10)
+    <<: *geosx_totalenergies_cluster_build
+    env:
+      - DOCKER_REPOSITORY=geosx/sherlock-gcc10.1.0-openmpi4.1.2-openblas0.3.10-zlib1.2.11
+      - CMAKE_BUILD_TYPE=Release
+      - BUILD_AND_TEST_ARGS=--disable-unit-tests
```

Comment:
Why do you disable them. They're KO?

Reply:
They use to time out in some linear Algebra tests. Will give them another try